test/testdata/container_config.json

```diff
@@ -58,6 +57,7 @@
 				"pid": 1
 			},
 			"readonly_rootfs": false,
+			"privileged": true,
```

Comment:
I think we should drop this one, and see which containers fail from not having priv